Source/Fuse.Reactive.JavaScript/TreeObservable/TreeObservable.uno

```diff
@@ -0,0 +1,201 @@
+using Uno;
+using Uno.Collections;
+using Fuse.Scripting;
+
+namespace Fuse.Reactive
+{
+	class TreeObservable: TreeObject, IMirror
+	{
+		public TreeObservable(Scripting.Object obj) : base(obj)
+		{
+			Set(this, obj);
+			Subscribe();
+		}
+
+		void Subscribe()
+		{
+			var obj = (Scripting.Object)Raw;
+			obj["$set"] = (Callback)Set;
```

Comment:
Is this adding the functions into the JS object. I saw calls to `$set` in one of the JS files. A terse explanation of this relationship would be helpful for future readers.